design/login.md

```diff
@@ -42,7 +42,7 @@ reading the `WWW-Authenticate` header DC/OS returns in the 401 response:
 {
   "dcos-oidc-auth0": {
     "authentication-type": "oidc-implicit-flow",
-    "client-method": "browser-prompt-authtoken",
+    "client-method": "browser-prompt-oidcidtoken-get-authtoken",
```

Comment:
This client method exists in our old Python CLI : https://github.com/dcos/dcos-core-cli/blob/a550652477175f48708a361fd0d65e21d243cded/python/lib/dcoscli/dcoscli/auth/main.py#L177